source/MaterialXGenShader/ShaderNode.h

```diff
@@ -215,14 +215,6 @@ class ShaderNode
         static const unsigned int SAMPLE2D    = 1 << 16; /// Can be sampled in 2D (uv space)
         static const unsigned int SAMPLE3D    = 1 << 17; /// Can be sampled in 3D (position)
         static const unsigned int CONVOLUTION2D = 1 << 18; /// Performs a convolution in 2D (uv space)
-
-        static const unsigned int COLOR_SPACE_TRANSFORM = 1 << 19; /// Performs color space transformation
-
-        static const unsigned int DO_NOT_OPTIMIZE = 1 << 20; /// Flag that this should not be optimized
-
-        static const unsigned int TRANSFORM_POINT = 1 << 21; /// A transform point node
-        static const unsigned int TRANSFORM_VECTOR = 1 << 22; /// A transform vector node
-        static const unsigned int TRANSFORM_NORMAL = 1 << 23; /// A transform normal node
```

Comment:
These classifications are not used anymore. It's leftover code from other changes.